file-tests/should-run/simpletest.fuse

```diff
@@ -0,0 +1,12 @@
+//let t: bit<16> =2;
```

Comment:
Can you remove this test since it exists in the FuTIL repository?

Reply:
Yeah so I also have to remove the test for matadd-fixed-points.fuse? Or just duplicate that for futil repository. Also I'll just push the updated version that supports arrays.

Reply:
I removed the simpletest and dot product/ fixed point array part is complete